lua/dashboard/session.lua

```diff
@@ -49,19 +53,34 @@ function session.session_load(name)
     api.nvim_command('mksession! ' .. fn.fnameescape(vim.v.this_session))
   end
 
-  if fn.filereadable(file_path) then
+  if fn.filereadable(file_path) > 0 then
     vim.cmd([[ noautocmd silent! %bwipeout!]])
     api.nvim_command('silent! source ' .. file_path)
 
     if vim.opt.laststatus:get() == 0 then
       vim.opt.laststatus = 2
     end
 
-    vim.notify('Loaded ' .. file_path .. ' session')
+    if db.session_verbose then
+      vim.notify('Loaded ' .. file_path .. ' session')
+    else
+      vim.notify('Session loaded')
+    end
     return
   end
 
-  vim.notify('The session ' .. file_path .. ' does not exist')
+  if db.session_verbose then
+    vim.notify('The session ' .. file_path .. ' does not exist')
+  else
+    vim.notify('No saved session for this directory')
+  end
+end
+
+function session.session_exists(name)
+  local file_name = name == nil and project_name() or name
+  local file_path = db.session_directory .. '/' .. file_name .. '.vim'
```

Comment:
don't hardcode the path separator . this only work for linux or mac. we should match the windows.

Reply:
you do the same thing in `session_save` and `session_load`, should I will fix those instances as well then? :)

Reply:
yup need remove the isWindows function use this

```lua
local is_windows = #vim.fn.windowsversion() > 0
local path_sep = is_windows and '\\' or '/'
```
then rewrite all the hardcode path sep

Reply:
I think I changed all occurences. Have a look.